README.md

```diff
@@ -20,14 +22,35 @@ To Do list: list structure
 
```

Comment:
Thank you for your suggestion, but can you kindly clarify it?
Because I know, that my live link shows only the content of the readme file. 
But as I know, it doesn't work for the webpack.

Reply:
Hi Vahan, 
that's fine as it is not a requirement, in the future you can consider using Netlify if you would like to add it.